src/components/PlatformFeatures/PlatformFeatures.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { Grid, View, Heading } from '@aws-amplify/ui-react';
+
+interface PlatformFeaturesProps {
+  platform: string;
+  children: React.ReactNode;
+}
+const PlatformFeatures: React.FC<PlatformFeaturesProps> = ({
+  platform,
+  children
+}) => {
+  return (
+    <Grid
+      templateColumns={{
+        base: '1fr',
+        medium: '1fr 1fr',
+        large: '1fr 1fr'
+      }}
+      gap="small"
+    >
+      <View columnSpan={2}>
+        <Heading
+          level={2}
+          fontWeight={'bold'}
+        >{`Features for ${platform}`}</Heading>
+      </View>
+      {children}
```

Comment:
Should the children fit into one column for the `base` screen sizes? When I adjust my browser window, it looks like there are still two columns instead of everything fitting into one:

![image](https://github.com/aws-amplify/docs/assets/54393192/2a7cfbb0-1314-4efa-bdbb-7d378b1bfb97)

Reply:
Good catch, will keep this  in mind! thank you.